### ISSUE TITLE
feature request

### ISSUE THREAD

**Issue by IntLucas** [PanieriLorenzo/vibe_machine#2]
Could tone thin out the sound at far right settings?

**Comment by PanieriLorenzo** [comment]
currently it is a simple 1-pole IIR lowpass filter, I can switch to a 1-pole IIR high-pass when turning the knob above a certain threshold, so it lowpasses below 50% and highpasses above. But wouldn't it be better if low-pass and highpass controls were separate?

**Comment by IntLucas** [comment]
Yes, you are probably right. I do like me some bandpassed delays.
